FWCore/Services/plugins/InitRootHandlers.cc

```diff
@@ -177,7 +177,9 @@ namespace {
        "nbinsy is <=0 - set to nbinsy = 1",
        "oneapi::tbb::global_control is limiting",
        "ufirst < fXmin, fXmin is used",
-       "ulast > fXmax, fXmax is used"}};
+       "ulast > fXmax, fXmax is used",
+       "The RNTuple file format will change.",
+       "Pre-release format version"}};
```

Comment:
Technically I think this shouldn't be necessary anymore with ROOT 6.34

Reply:
You are correct.